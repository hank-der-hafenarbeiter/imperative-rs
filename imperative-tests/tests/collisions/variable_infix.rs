use imperative_rs::InstructionSet;
#[derive(InstructionSet)]
enum Instructionset {
    #[ opcode = "0xff_ff_ff" ]
    A,
    #[ opcode = "0xff_vv_ff" ]
    B{v:u8},
}

fn main() {}
