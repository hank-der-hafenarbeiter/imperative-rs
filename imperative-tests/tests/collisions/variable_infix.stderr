error: Opcode collides with other opdcode
 --> $DIR/variable_infix.rs:6:17
  |
6 |     #[ opcode = "0xff_vv_ff" ]
  |                 ^^^^^^^^^^^^

error: Collides with this opcode
 --> $DIR/variable_infix.rs:4:17
  |
4 |     #[ opcode = "0xff_ff_ff" ]
  |                 ^^^^^^^^^^^^
